packages/utils/index.ts

```diff
@@ -144,16 +144,15 @@ export function capitalize(word: string) {
 export const addressRegex = /0x[a-fA-F0-9]{40}$/;
 
 /**
- * Formats a date in the "MM/DD/YYYY" (American) format.
+ * Formats a date in the "Month Date, Year" (American) format.
  *
  * @param {Date} date - The input date to be formatted.
  * @returns {string} A string representing the formatted date.
  */
 export function formatDate(date: Date): string {
   return date.toLocaleDateString("en-US", {
-    month: "2-digit",
     day: "2-digit",
+    month: "short",
     year: "numeric",
-    timeZone: "UTC",
```

Comment:
have you checked whether this might impact elsewhere?

Reply:
yes, this was being used on `RoundOverviewCards` and `PoolOverviewCards`. But the intention here is to impact those places hahahahha

They were showing the wrong end date, which is always wed 21:00. 31th is a Thursday. So the round select is showing the correct end date

<img width="399" alt="image" src="https://github.com/bleu-studio/balancer-tools/assets/63298341/6e5f2f19-9e6c-470a-8273-b76a504680be">

Reply:
Wed 21h UTC-3 right? But isn't the end date Wed Wed 23:59:59 UTC?

Reply:
🤔 hmmmm yeah kkkkkkk that makes sense, but then `rounds.ts` weren't considering the timezone... then by definition the round ends at 00:00 UTC Thursday, correct?

Reply:
added the timezone back and added this to help the user understand this date too. Just wondering if we should just adjust according to the browser timezone 🤔 

<img width="413" alt="image" src="https://github.com/bleu-studio/balancer-tools/assets/63298341/59d3ac8f-3f7d-4161-b9f6-50aa08cd150f">

Reply:
No, let's always deal with UTC only.

Reply:
> 🤔 hmmmm yeah kkkkkkk that makes sense, but then `rounds.ts` weren't considering the timezone... then by definition the round ends at 00:00 UTC Thursday, correct?

It does:
```
// rounds.ts
... // #2 below
  public static FIRST_ROUND_END_DATE = new Date("2022-04-14T00:00:00.000Z");
```

https://en.wikipedia.org/wiki/ISO_8601 see "Date and time in UTC"